lib/weapon.rb

```diff
@@ -0,0 +1,24 @@
+class Weapon
+  attr_reader :choice
+  CHART = {
+    rock: [:scissors, :lizard],
+    paper: [:rock, :spock],
+    scissors: [:paper, :lizard],
+    lizard: [:paper, :spock],
+    spock: [:rock, :scissors]
+  }
+
+  def initialize(choice = random_choice)
+    @choice = choice.to_sym
+  end
+
```

Comment:
This is a clever solution!

Reply:
Methods that return a boolean should end in a `?`

Reply:
How about this alternative? https://github.com/EdwardAndress/rps-weapons

It's kind of a novelty solution. I.e. I'm not sure it's 'good' but you might find it interesting!